f8a_worker/workers/stackaggregator_v2.py

```diff
@@ -86,21 +84,42 @@ def extract_component_details(component):
         "osio_user_count": component.get("osio_user_count", -1),
         "latest_version": latest_version,
         "github": github_details,
-        "code_metrics": code_metrics
+        "code_metrics": code_metrics,
     }
-    return component_summary, licenses
+    return component_summary
 
 def aggregate_stack_data(stack, manifest_file, ecosystem, deps):
     dependencies = []
     licenses = []
+    license_score_list = []
     for component in stack.get('result', []):
         data = component.get("data", None)
         if data:
-            component_data, component_licenses = extract_component_details(data[0])
+            component_data = extract_component_details(data[0])
+            # create license dict for license scoring
+            license_scoring_input = {
+                'package': component_data['name'],
+                'version': component_data['version'],
+                'license': component_data['licenses']
+            }
             dependencies.append(component_data)
-            licenses.extend(component_licenses)
+            licenses.extend(component_data['licenses'])
+            license_score_list.append(license_scoring_input)
 
     stack_distinct_licenses = set(licenses)
+    # Call License Scoring to Get Stack License
+    license_scoring_url = "http://stack-license-stack-license.dev.rdu2c.fabric8.io/api/v1/stack_license"
```

Comment:
This is the hard-coded url for license-scoring. @rouseguy will have to take it to the next level of setting env variables and stuff.